packaging/helm-charts/helm3/strimzi-drain-cleaner/values.yaml

```diff
@@ -52,6 +57,14 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
+securityContext: {}
+  # Specifies optional security context, depending your environment prerequisites.
+  # If you want to specify security contexts add them below and remove curly braces above.
+
+labels: {}
+  # Specifies optional custom labels for Deployment and Pod.
+  # If you want to add custom labels, add them below and remove curly braces above.
```

Comment:
That's because my company requires specific labels on both Deployment and Pods for support and finops purposes (things like: owner/contact, project, ...) and unfortunately existing labels do not provide these details.

Reply:
Well, but others might not want the samelabels for both. So I think having separate options with a bit more clear naming of what they do should be used? E.g. `podLabels` and `deploymentLabels`? 

(I also wonder if it isn't just easier to use the YAML files for installation instead of having a Helm Chart with tens of different customizations because everyone wants something different. But I guess that is expected from someone who does not use Helm in the first place 🤷)

Reply:
Good point (even if I am not 100% convinced). I push an update.